mopidy/backend.py

```diff
@@ -250,7 +263,9 @@ def change_track(self, track):
             logger.debug("Backend translated URI from %s to %s", track.uri, uri)
         if not uri:
             return False
-        self.audio.set_uri(uri, live_stream=self.is_live(uri)).get()
+        self.audio.set_uri(
+            uri, live_stream=self.is_live(uri), download=self.should_download(),
```

Comment:
Could we not log it right here?

Reply:
`change_track()` can be implemented by extensions, so maybe not much point logging here.